modules/gcp_bigquery_big_lake_table/inputs.tf

```diff
@@ -35,12 +37,18 @@ variable "hive_partitioning_mode" {
     condition     = contains(["AUTO", "STRINGS", "CUSTOM"], var.hive_partitioning_mode)
     error_message = "Source format of table must be NEWLINE_DELIMITED_JSON, AVRO or PARQUET"
   }
+  default = "AUTO"
 }
 
 variable "hive_source_uri_prefix" {
   type        = string
   description = "A common for all source uris must be required. The prefix must end immediately before the partition key encoding begins"
+  validation {
+    condition     = !endswith(var.hive_source_uri_prefix, "/") && !endswith(var.hive_source_uri_prefix, "*")
+    error_message = "hive_source_uri_prefix should not end with a `/` or `*`"
```

Comment:
Why prefixes ending with `/` are disallowed. From [Terraform docs](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_table#hive_partitioning_options):
> When hive partitioning is requested with either AUTO or STRINGS detection, the common prefix can be either of `gs://bucket/path_to_table` or `gs://bucket/path_to_table/`.

This is also breaking our test suite so if we set this limitation, we need to fix the test suite as well

Reply:
Yeah I saw the failing test, I can allow it if I use trim in the `uri_prefixes`